generators/app/templates/_package.json

```diff
@@ -3,30 +3,31 @@
   "version": "0.0.0-semantically-released",
   "description": "<%= description %>",
   "main": "dist/index.js",
-  "engines": {
-    "node": ">=10",
-    "npm": ">=6",
-    "yarn": ">=1"
+  "keywords": [],
+  "author": "Kent C. Dodds <kent@doddsfamily.us> (http://kentcdodds.com)",
```

Comment:
Could we update my email address to kentcdodds@gmail.com and let's make the website `https`